src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/Server/Properties/launchSettings.json

```diff
@@ -14,14 +14,14 @@
   "profiles": {
     "IIS Express": {
       "commandName": "IISExpress",
-      "launchBrowser": true,
+      "launchBrowser": false,
```

Comment:
We don't need to launch the API since the client project already launches the browser.